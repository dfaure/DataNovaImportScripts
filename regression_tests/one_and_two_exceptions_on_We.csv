00049A;BOURG EN BRESSE FONTAINE BERNARD BP;2021-08-22;FERME;;;;;;;;;;;FERME;;;;;;;
00049A;BOURG EN BRESSE FONTAINE BERNARD BP;2021-08-23;FERME;;;;;;;;;;;FERME;;;;;;;
00049A;BOURG EN BRESSE FONTAINE BERNARD BP;2021-08-24;09:00-12:30;13:30-17:30;;;;;;16:00;16:00;16:00;;09:00-12:30;13:30-17:30;;;;;;
00049A;BOURG EN BRESSE FONTAINE BERNARD BP;2021-08-25;09:00-12:30;;;;;;;12:30;12:30;12:30;;09:00-12:30;13:30-17:30;;;;;;
00049A;BOURG EN BRESSE FONTAINE BERNARD BP;2021-08-26;09:00-12:30;13:30-17:30;;;;;;16:00;16:00;16:00;;09:00-12:30;13:30-17:30;;;;;;
00049A;BOURG EN BRESSE FONTAINE BERNARD BP;2021-08-27;09:00-12:30;13:30-17:30;;;;;;16:00;16:00;16:00;;09:00-12:30;13:30-17:30;;;;;;
00049A;BOURG EN BRESSE FONTAINE BERNARD BP;2021-08-28;FERME;;;;;;;;;;;FERME;;;;;;;
00049A;BOURG EN BRESSE FONTAINE BERNARD BP;2021-08-29;FERME;;;;;;;;;;;FERME;;;;;;;
00049A;BOURG EN BRESSE FONTAINE BERNARD BP;2021-08-30;FERME;;;;;;;;;;;FERME;;;;;;;
00049A;BOURG EN BRESSE FONTAINE BERNARD BP;2021-08-31;09:00-12:30;13:30-17:30;;;;;;16:00;16:00;16:00;;09:00-12:30;13:30-17:30;;;;;;
00049A;BOURG EN BRESSE FONTAINE BERNARD BP;2021-09-01;09:00-12:30;13:30-17:30;;;;;;16:00;16:00;16:00;;09:00-12:30;13:30-17:30;;;;;;
00049A;BOURG EN BRESSE FONTAINE BERNARD BP;2021-09-02;09:00-12:30;13:30-17:30;;;;;;16:00;16:00;16:00;;09:00-12:30;13:30-17:30;;;;;;
00049A;BOURG EN BRESSE FONTAINE BERNARD BP;2021-09-03;09:00-12:30;13:30-17:30;;;;;;16:00;16:00;16:00;;09:00-12:30;13:30-17:30;;;;;;
00049A;BOURG EN BRESSE FONTAINE BERNARD BP;2021-09-04;09:00-12:30;;;;;;;12:00;12:00;12:00;;09:00-12:30;;;;;;;
00049A;BOURG EN BRESSE FONTAINE BERNARD BP;2021-09-05;FERME;;;;;;;;;;;FERME;;;;;;;
00049A;BOURG EN BRESSE FONTAINE BERNARD BP;2021-09-06;FERME;;;;;;;;;;;FERME;;;;;;;
00049A;BOURG EN BRESSE FONTAINE BERNARD BP;2021-09-07;09:00-12:30;13:30-17:30;;;;;;16:00;16:00;16:00;;09:00-12:30;13:30-17:30;;;;;;
00049A;BOURG EN BRESSE FONTAINE BERNARD BP;2021-09-08;09:00-12:30;13:30-17:30;;;;;;16:00;16:00;16:00;;09:00-12:30;13:30-17:30;;;;;;
00049A;BOURG EN BRESSE FONTAINE BERNARD BP;2021-09-09;09:00-12:30;13:30-17:30;;;;;;16:00;16:00;16:00;;09:00-12:30;13:30-17:30;;;;;;
00049A;BOURG EN BRESSE FONTAINE BERNARD BP;2021-09-10;09:00-12:30;13:30-17:30;;;;;;16:00;16:00;16:00;;09:00-12:30;13:30-17:30;;;;;;
00049A;BOURG EN BRESSE FONTAINE BERNARD BP;2021-09-11;09:00-12:30;;;;;;;12:00;12:00;12:00;;09:00-12:30;;;;;;;
