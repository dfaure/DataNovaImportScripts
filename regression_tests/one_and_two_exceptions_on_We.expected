00049A|BOURG EN BRESSE FONTAINE BERNARD BP|Tu-Fr 09:00-12:30,13:30-17:30; Sa 09:00-12:30; PH off; 2021 Aug 25 09:00-12:30; 2021 Aug 28 off
